README.md

```diff
@@ -0,0 +1,6 @@
+1. Copy your msso_config.json file inside platforms/android/assets folder
```

Comment:
1. Add your msso_config.json file to the folder: platforms/android/assets.
 2. Copy the MAS libraries to the folder: platforms/android/libs. Name the libraries as shown in the build.gradle file.
 +
 +```
 +Disclaimer: This app is built using Cordova version 7.1.0. The app will not compile with Cordova Version 8.0, because Cordova Android changed the internal structure of Android folders.